### PR TITLE
Remove the Id property from ModelGroup

### LINKED ISSUE #413: Id property in ModelGroup is not intuitive and likely not needed
We could remove `Id` completely, and instead use the enum value of `ModelGroupName`.

Otherwise `Id` should be renamed to something more descriptive, like `ReadmeIndex`.

### PR DESCRIPTION
**Pull request by stevk**
Changes made to the manifest code and ModelGroupName enum were done to keep the existing order intact. Also removed a String.Format that hadn't been updated yet.

Fixes #413

### CODE REVIEW

File: Source/ModelGroup.cs [hunk 1]
```diff
@@ -13,7 +13,6 @@ internal abstract partial class ModelGroup
         public List<Property> Properties { get; private set; }
         public List<Runtime.Image> UsedTextures { get; private set; }
         public List<Runtime.Image> UsedFigures { get; private set; }
-        public int Id { get; set; }
```
Comment (bghgary) [inline Source/ModelGroup.cs]: I think we should consider changing `ModelGroupName` to `ModelGroupId` and change the property `Name` to `Id`

File: Source/ReadmeBuilder.cs [hunk 2]
```diff
@@ -121,7 +121,7 @@ public void SetupTable(ModelGroup test, int modelIndex, List<Property> model)
         {
             string modelGroupName = test.Name.ToString();
             string modelNumber = modelIndex.ToString("D2");
-            string liveURL = string.Format("https://bghgary.github.io/glTF-Assets-Viewer/?folder={0}&model={1}", test.Id, modelIndex);
+            string liveURL = $"https://bghgary.github.io/glTF-Assets-Viewer/?folder={Convert.ToInt32(test.Name)}&model={modelIndex}";
```
Comment (bghgary) [inline Source/ReadmeBuilder.cs]: `{Convert.ToInt32(test.Name)}` can be `{test.Name:D}`

File: Source/ModelGroup.cs (hunk 1 above)
Comment (stevk) [reply Source/ModelGroup.cs]: I thought the term `Id` was somewhat ambiguous in this context, and the enum is used as a string in quite a few more places than it is used as an int. Is there a strong argument for `Id`?

Comment (bghgary) [reply Source/ModelGroup.cs]: The word `name` is confusing to me as an enum. `name` is typically a string.

File: Source/Manifest.cs [hunk 3]
```diff
@@ -52,16 +52,16 @@ private static class CustomCameraList
             public class ModelCameraPairing
             {
                 public Camera camera;
-                public ModelGroupName modelGroup;
+                public ModelGroupId? modelGroup;
 
-                public ModelCameraPairing(Camera cameraSettings, ModelGroupName name)
+                public ModelCameraPairing(Camera cameraSettings, ModelGroupId? name)
```
Comment (bghgary) [inline Source/Manifest.cs]: nit: name -> modelGroup

File: Source/ReadmeBuilder.cs [hunk 4]
```diff
@@ -161,7 +161,7 @@ public void SetupTable(ModelGroup test, int modelIndex, List<Property> model)
         public void WriteOut(Assembly executingAssembly, ModelGroup test, string assetFolder)
         {
             string template;
-            string templatePath = $"AssetGenerator.ReadmeTemplates.{test.Name.ToString()}.md";
+            string templatePath = $"AssetGenerator.ReadmeTemplates.{test.Id.ToString()}.md";
```
Comment (bghgary) [inline Source/ReadmeBuilder.cs]: `.ToString()` is extraneous

Comment (stevk) [reply Source/ReadmeBuilder.cs]: Fixed

**Pull request merged**